roles/sriov_config/tasks/check_sriov_network.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Wait until NetworkAttachmentDefinition is created from SriovNetwork {{ sriov_conf.network.name }}
+  community.kubernetes.k8s_info:
+    api_version: k8s.cni.cncf.io/v1
+    name: "{{ sriov_conf.network.name }}"
+    namespace: "{{ sriov_conf.network.network_namespace }}"
+    kind: NetworkAttachmentDefinition
+  register: net_attach_def_check
+  retries: 6
+  delay: 10
+  until: net_attach_def_check.resources|length == 1
+  no_log: true
```

Comment:
should we remove no_log here to ease the debug?

Reply:
Maybe @tonyskapunk can clarify on this, but I assume the `no_log` is placed here to avoid disclosing network information that may be sensible, right?
Honestly, the output is not really throwing relevant information, it's just saying that the resource is created and its specification can be found in the variable holding the SRIOV config, so maybe it's not a big deal to remove the `no_log`

Reply:
I believe no_log was to remove the bulky logs here, we sometimes use it by default. But here the logs are not that bulky I guess and could really help when debugging

Reply:
For the moment, if you agree, I'll merge this, since I have other changes depending on this that I need to merge, but we can explore this in a future change, in case we need it.
But I'd say we're hiding here the logs to avoid exposing net info that may be sensible, that's my assumption. Anyway let's revisit it in a new change if required